extensions/ql-vscode/package.json

```
@@ -295,6 +296,10 @@
         "command": "codeQL.openReferencedFile",
         "title": "CodeQL: Open Referenced File"
       },
+      {
+        "command": "codeQL.previewQueryHelp",
+        "title": "CodeQL: Preview Query Help File"
```

Minor. I think this is clear enough without having to specify it's a file.
```suggestion
        "title": "CodeQL: Preview Query Help"
```